### DISCUSSION TITLE
Show none or Pre-Select table row

### DISCUSSION THREAD

**Discussion by TheVern** [EvotecIT/PSWriteHTML#327] [Q&A]
Hey there - 

Really enjoying playing around with this incredible project.

My question is as follows:

I have 2 datatables, and have a TableEvent linked from table 1 to table 2.

The idea is to filter table 2 on specific values from table1. This works really well... However on initial load, I want table2 to display nothing until a row is selected in table1. At the moment table1 and table 2 shows all records until a selection is made in Table1

(Hope this makes sense)

Or optionally, if I can pre-select a record in table1 on load to only display that linked item in table2 - it will be acceptable 

greatly looking forward to your reply

Thank you

**Comment by PrzemyslawKlys** [comment]
While it's not obvious you could just search for something that doesn't exists in the table

```powershell
New-HTMLTable -DataTable $DataTable1 -Title 'Test1' -PagingLength 2 -Find 'Something' 
```

Probably searching for "     " would work making the 2nd table empty.

**Reply by TheVern** [reply]
Hi - Thanks for your suggestion

When doing this it does not filter the table and shows all values

When doing a find on something else like "bob" the table is empty as no results are found, But you then have to remove that from the search bar before the table reacts

**Reply by PrzemyslawKlys** [reply]
I guess it's probably because of Find using searchbox, while the connection between 2 tables using filtering on specific column. To fix this a new feature would need to be developed.

**Reply by StefTan** [reply]
Hello, I have the same problem. On reload the old (in previous sessions) setted filters where applied to the tables I linked. Is there w way to start with empty filters. Killing the browser session is not helping a new session will come up wit sam filters (set by the previous sync. Using -filter option bij the Table creation will display those settings and you can wipe them.

**Reply by PrzemyslawKlys** [reply]
Please open issues/rather then discussions. If you believe this is a bug.
